lib/utilityFunctions.js

```diff
@@ -24,27 +31,29 @@ const returnRest = (val, input, rest, field) => {
     }
   }
   output.str = rest
-  val.cursorLoc = {line: output.line, column: output.column}
+  val.cursorLoc = { line: output.line, column: output.column }
   return [val, output]
 }
 
-const isEmptyObj = obj => {
-  for (let prop in obj) {
-    if (obj.hasOwnProperty(prop)) return false
+const isEmptyObj = (obj) => {
+  for (const prop in obj) {
+    if (Object.prototype.hasOwnProperty.call(obj, prop)) return false
```

Comment:
This is a nice catch, maybe talk about why this is preferred to `obj.hasOwnProperty` this Friday @sumanthyedoti